explain/setup/heroku/03_review_settings.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Step 3: Review auto_explain settings'
+install_track_title: 'Automated EXPLAIN: Setup (Heroku Postgres)'
+backlink_href: /docs/explain/setup
+backlink_title: 'Automated EXPLAIN: Setup'
+---
+
+import PgSettingsRecommendations, { getAutoExplainSettingRecommendations } from "../../../components/PgSettingsRecommendations";
+
+export const AutoExplainLink = ({version, children}) => {
+  return (
+    <a href={`https://www.postgresql.org/docs/${version ?? 'current'}/auto-explain.html`} target='_blank'>
+      {children}
+    </a>
+  )
+}
+
+There are a number of `auto_explain` settings that allow you to configure which queries
+will generate EXPLAIN plans. You can read the full Postgres documentation about
+these <AutoExplainLink version={props.pgMajorVersion}>here</AutoExplainLink>.
+The right configuration will depend on your server and workload,
+but we've found the following is a good starting point:
+
+<PgSettingsRecommendations recommendations={getAutoExplainSettingRecommendations(props.settings)} />
+
+Heroku offers to modify several of these settings, but not all of them. Notably,
+you cannot update the `auto_explain.log_format` setting with Heroku Postgres
+currently. Only JSON format is supported by pganalyze and as text format support
+is limited, you may see the errors on some EXPLAIN plans.
```

Comment:
I think "only JSON format is supported" may be a little strong. Text format is definitely a second-class citizen, but it mostly works, especially if you're not doing anything too weird in your queries. I've used the phrase "best-effort" to describe this in the past, so maybe

```suggestion
currently. Text-format auto_explain output is supported on a best-effort basis, so you may see errors for some EXPLAIN plans.
```

Reply:
Yeah agree with you that "only JSON format is supported" is a little strong (but we're actually using it in a tooltip 🙃). Given that other providers can change this config, I think we are fine leaving the tooltip as is, and update here as you suggested (I like the best-effort, as we are definitely doing the best effort there)